app/Providers/HelperServiceProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
```

Comment:
link for `app/Helpers/helpers.php` is already in `https://github.com/HDRUK/gateway-api/blob/649e59ec123174bfa8be3287db792328347f5b4b/composer.json#L76`.
do you need this ?

Reply:
Yes, because at the point of `db:seed` composer isn't autoloaded. If you completely wipe your clone, and start fresh you cannot migrate/seed from a fresh clone because of the usage of functions that haven't been loaded because app hasn't been bootstrapped in the same way.